maintainers/maintainer-list.nix

```diff
@@ -14720,4 +14720,10 @@
     github = "dfithian";
     githubId = 8409320;
   };
+  nsnelson = {
+    email = "noah.snelson@protonmail.com";
+    name = "Noah Snelson";
+    github = "peeley";
+    githubId = 30942198;
+  };
```

Comment:
Please sort this alphabetical

Reply:
Done!

Reply:
you reverted it when force pushing

Reply:
Whoops, my bad! For some reason, I thought you meant alphabetizing the fields of the `nsnelson` map 😅. Should be fixed now.